samples/todoapp/src/test/java/com/examples/todoapp/tasks/TaskListFormulaTest.kt

```diff
@@ -4,24 +4,21 @@ import com.examples.todoapp.data.Task
 import com.examples.todoapp.data.TaskRepo
 import com.google.common.truth.Truth.assertThat
 import com.instacart.formula.test.test
-import com.nhaarman.mockito_kotlin.mock
-import com.nhaarman.mockito_kotlin.whenever
 import io.reactivex.rxjava3.core.Observable
 import org.junit.Test
 
 class TaskListFormulaTest {
 
-    @Test fun `change filter type`() {
-        val repo = mock<TaskRepo>()
-        whenever(repo.tasks()).thenReturn(Observable.just(
-            listOf(
-                Task("Mow the lawn."),
-                Task("Go get a haircut.")
-            )
-        ))
+    private val showToast = { toast: String ->
+
+    }
+
+    @Test
+    fun `change filter type`() {
```

Comment:
I wasn't sure if we wanted to keep this test around, it doesn't currently run in CI